src/cread.c

```diff
@@ -19,105 +19,155 @@
 #include "cread.h"
 
 /**
- * This subroutine opens a new system file for reading BUFR messages.
+ * Open a new system file for reading BUFR messages.
  *
- * @param[in] nfile -- f77int*: Internal Fortran I/O stream index associated with ufile
- * @param[in] ufile -- char*: [path/]name of system file to be opened
+ * @param nfile - File ID.
+ * @param ufile - [path/]name of system file to be opened.
  *
  * @author J. Woollen @date 2012-09-15
  */
-void openrb   (nfile,ufile) f77int *nfile; char *ufile; { pb[*nfile] = fopen( ufile , "rb " ); }
+void openrb( int nfile, char *ufile ) {
```

Comment:
re: anchoring the function to the first column, I wasn't aware that was a common convention.  I don't mind doing that, but you'll have to give me a few minutes to push up another commit, because there's quite a bit of C code in the library.

re: the spacing before and after the parentheses, I like them for readability.  But you probably have more experience in the C world than me, so I'll defer to your judgement and if you think that's also an oddball feature then I'll take those out too.